### PR TITLE
Assign reviewer for github PRs

### PR DESCRIPTION
**Pull request by Onager**
Assign reviewer for github PRs

### CODE REVIEW

File: l2tdevtools/review_helpers/github.py [hunk 1]
```diff
@@ -92,6 +92,34 @@ def CreatePullRequestReview(
 
     return True
 
+  def AssignPullRequest(
+      self, pull_request_number, access_token, assignees):
+    """Assigns a GitHub pull request.
+
+    Args:
+      pull_request_number (int): GitHub issue number of the pull request.
+      access_token (str): github access token.
```
Comment (joachimmetz) [inline l2tdevtools/review_helpers/github.py]: "github" => "GitHub"

File: l2tdevtools/review_helpers/github.py [hunk 2]
```diff
@@ -92,6 +92,34 @@ def CreatePullRequestReview(
...
+      assignees (list[str]): github usernames to assign.
+
+    Returns:
+      bool: True if the review was created.
```
Comment (joachimmetz) [inline l2tdevtools/review_helpers/github.py]: => "True if the assignees were successfully added."

File: l2tdevtools/review_helpers/github.py [hunk 3]
```diff
@@ -92,6 +92,34 @@ def CreatePullRequestReview(
 
     return True
 
+  def AssignPullRequest(
+      self, pull_request_number, access_token, assignees):
+    """Assigns a GitHub pull request.
```
Comment (joachimmetz) [inline l2tdevtools/review_helpers/github.py]: => "Adds assignees to a GitHub pull request." or "Sets the assignees to a GitHub pull request."

The role of an assignee is to ....

File: tests/review_helpers/github.py [hunk 4]
```diff
@@ -60,6 +60,16 @@ def testRequestPRReview(self):
 
     self.assertTrue(result)
 
+  def testAssignPullRequest(self):
+    """Tests the RequestPullReview function."""
```
Comment (joachimmetz) [inline tests/review_helpers/github.py]: RequestPullReview => AssignPullRequest

File: l2tdevtools/review_helpers/github.py (hunk 3 above)
Comment (Onager) [reply l2tdevtools/review_helpers/github.py]: Done

File: l2tdevtools/review_helpers/github.py (hunk 1 above)
Comment (Onager) [reply l2tdevtools/review_helpers/github.py]: Done

File: l2tdevtools/review_helpers/github.py (hunk 2 above)
Comment (Onager) [reply l2tdevtools/review_helpers/github.py]: Done

File: tests/review_helpers/github.py (hunk 4 above)
Comment (Onager) [reply tests/review_helpers/github.py]: Done.

**Pull request merged**